packages/@icon-magic/generate/test/fixtures/out/home-2/ads/build/iconrc.json

```diff
@@ -37,12 +37,14 @@
       "name": "active-small",
       "path": "./active-small.svg",
       "imageset": "active-small",
+      "colorScheme": "light",
```

Comment:
Do we have to have this? Can it be assumed to default to `light`, if there's a corresponding asset with the same imageset that's `dark`?

Reply:
we don't, we can assume as you mentioned. it's not actually used for anything at this point, I just did here as an example.